docs/source/evennia_core/system/django/serverconf.md

```diff
@@ -0,0 +1,4 @@
+## ServerConf
```

Comment:
this is deep deep stuff, probably should be its own page and not ever really mentioned in the rest of the docs. people can find it in the toctree if they want it.